vendor/engines/split_accounts/app/models/split_accounts/split_account.rb

```diff
@@ -25,10 +26,23 @@ def one_split_has_extra_penny
       end
     end
 
+    def unique_split_subaccounts
+      if duplicate_subaccounts?
+        errors.add(:splits, :duplicate_subaccounts)
+      end
+    end
+
     def extra_penny_count
       splits.select(&:extra_penny?).size
     end
 
+    def duplicate_subaccounts?
+      splits.map(&:subaccount_id)
```

Comment:
It's not that bad, but it seems like there should be a cleaner way to write this.

```
subaccounts = splits.map(&:subaccount_id)
subaccounts.length == subaccounts.uniq.length
```
or
```
splits.map(&:subaccount_id)
  .group_by { |e| e }
  .any? { |k, v| v.size > 1 }
```